src/pages/endpoint/index.tsx

```diff
@@ -195,7 +195,7 @@ const Endpoint = () => {
                                     type="text"
                                     label="Server URL"
                                     background="white"
-                                    placeholder={'e.g. green.binaryws.com'}
+                                    placeholder={'e.g. green.derivws.com'}
```

Comment:
pls make it `placeholder="e.g. green.derivws.com"`

Reply:
Sure @dan-deriv  but may I know why? What is the difference and how it will have an impact?

Reply:
Moreover, it is semantically misleading.

Reply:
@dan-deriv I'm getting confused here I couldn't find any reference for the things you mentioned and as far as I checked there is not much difference between the way you mentioned and currently implemented one  it's just the syntax ( which will be personal prefrences),  if you can help to refer me to a refrence I can read and learn I will be thankful